app/assets/stylesheets/application.scss

```diff
@@ -83,3 +85,25 @@ body {
   font-size: 10pt;
   margin-top: 11px;
 }
+
+.player-list {
+  margin: 20px;
+  display: inline-flex;
+}
+
+ul .player-panel {
+  margin: 6px;
+  padding: 20px 10px 20px 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `20px 10px` instead of `20px 10px 20px 10px`